### PR TITLE
Export build and test metadata

### PR DESCRIPTION
**Pull request by sshyvr**
Export build and test metadata

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,6 +44,24 @@ const promptSchema = {
...
+          type: 'boolean',
+          message: 'include build metadata (true/false)',
+          required: false,
+          default: false
+        },
+        includeTestResult: {
+            type: 'boolean',
+            message: 'include test results for each build (true/false)',
+            required: false,
+            default: true
```
Comment (richiebres) [inline index.js]: I would default to `false` here

File: index.js [hunk 2]
```diff
@@ -44,6 +44,24 @@ const promptSchema = {
...
+        },
+        includeTestCoverage: {
+            type: 'boolean',
+            message: 'include test coverage for each build (true/false)',
+            required: false,
+            default: true
```
Comment (richiebres) [inline index.js]: I would default to `false` here too

File: index.js (hunk 1 above)
Comment (sshyvr) [reply index.js]: Updated to `false`

File: index.js (hunk 2 above)
Comment (sshyvr) [reply index.js]: Done - updated this one as well.

**Pull request merged**